Dockerfile

```diff
@@ -37,15 +37,18 @@ RUN export DATABASE_URL=mysql2://localhost/temp && \
     apt-get install -y mysql-server && \
     /etc/init.d/mysql start && \
     cp config/app_config.yml.SAMPLE config/app_config.yml && \
+    cp config/database.yml.MySQL_SAMPLE config/database.yml && \
```

Comment:
We have `DATABASE_URL`, right - why don't we use that?

Reply:
this way we can use `multicoops:run` again, which becomes very handy for migrations

Reply:
That makes sense, good!